nfs-ganesha/config/definitions/nfs-ganesha.yml

```diff
@@ -105,6 +108,14 @@ If this is checked, then the binaries will be built and pushed to chacra even if
           name: ARCH
           values:
             - ARCHS
+    triggers:
+      - github
+      - pollurl:
+          urls:
+            - url: https://shaman.ceph.com/api/repos/ceph/master/latest/centos/7/repo
```

Comment:
@andrewschoen added it, do you think i need to add the "text" content type underneath simple when the SHA1's change?